godtools/Views/TractElements/BaseTractElement.swift

```diff
@@ -197,12 +197,12 @@ class BaseTractElement: UIView {
     
     func getPreviousElement() -> BaseTractElement? {
         guard let index = getElementPosition() else {
-            return self.elements?.last
+            return self.parent?.elements?.last
```

Comment:
The answer is the same, but, there was a logical/dumb problem before, it was trying to get the previous element on the childrens elements (really dumb, sorry...) so now instead it is searching on the parent elements